src/main/kotlin/com/seokjoo/todo/domain/entity/todouser/TodoUser.kt

```diff
@@ -0,0 +1,19 @@
+package com.seokjoo.todo.domain.entity.todouser
+
+import com.seokjoo.todo.domain.entity.BaseEntity
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+
+@Entity
+class TodoUser(
```

Comment:
```suggestion
class User(
```

그냥 User는 어떤가요!
TodoUser는 먼가 Todo랑 User의 중간 테이블로 생각할 수도 있을 것 같아요 🤔

Reply:
헉 좋아요 근데, 기본 user로 네이밍된 테이블은 못만들지 않나요?! mysql 내부에도 user가 있어서요 ! 
클래스명을 User로 하고, 테이블 네임만 변경해볼까용 ?

Reply:
헉 그러네요
mysql에서 user가 예약어여서 ...!

> 클래스명을 User로 하고, 테이블 네임만 변경해볼까용 ?

조아요 !!

Reply:
[ca871bc](https://github.com/Ahn-seokjoo/todo-spring/pull/39/commits/ca871bc991c2cda093052169bf9a8aa7b11b2b7f) 완료~

Reply:
fyi, @Table(name="`user`") 요렇게 하면 (백틱을 넣으면) user를 테이블명으로 쓸 수도 있을거에여!

Reply:
@sysnar 좀 늦게봤네용 
저렇게되면 MySQL에서 이미 만들어져있던 user 테이블은 어떻게돼요 ?!

Reply:
MySQL 만들자마자 비밀번호랑 같이 관리되는
기본 테이블인 user 가 있는거 맞죠 ?!

Reply:
@Ahn-seokjoo 
백틱을 넣어서 테이블을 생성하면 기존 예약어로 등록된 테이블과는 다른 테이블이 생성돼요!
```
`user` != user
```
그래서 쿼리를 할때에도 아래처럼 해야할 거에요.
(jpa에서는 자동으로 해줌!)

```sql
select * from `user`;
```

Reply:
오홍 꿀팁 배워가요